queries.sql

```diff
@@ -58,48 +58,113 @@ SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1
 
 
 
--- What animals belong to Melody Pond?
 SELECT animals.name
 FROM animals
 JOIN owners ON animals.owner_id = owners.id
 WHERE owners.full_name = 'Melody Pond';
  
-
--- List of all animals that are pokemon (their type is Pokemon)
 SELECT animals.name
 FROM animals
 JOIN species ON animals.species_id = species.id
 WHERE species.name = 'Pokemon';
 
--- List all owners and their animals, remember to include those that don't own any animal
 SELECT owners.full_name, animals.name
 FROM owners
 LEFT JOIN animals ON owners.id = animals.owner_id;
 
--- How many animals are there per species?
 SELECT species.name, COUNT(*) AS count
 FROM animals
 JOIN species ON animals.species_id = species.id
 GROUP BY species.name;
 
--- List all Digimon owned by Jennifer Orwell
 SELECT animals.name
 FROM animals
 JOIN species ON animals.species_id = species.id
 JOIN owners ON animals.owner_id = owners.id
 WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
 
--- List all animals owned by Dean Winchester that haven't tried to escape
 SELECT animals.name
 FROM animals
 JOIN owners ON animals.owner_id = owners.id
 WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
 
--- SELECT owners.full_name, COUNT(*) AS count
 SELECT owners.full_name, COUNT(*) AS count
 FROM animals
 JOIN owners ON animals.owner_id = owners.id
 GROUP BY owners.full_name
 ORDER BY count DESC
 LIMIT 1;
 
+
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.name AS last_animal_seen
+FROM visits
+JOIN animals ON visits.animal_id = animals.id
+WHERE visits.vet_id = (SELECT id FROM vets WHERE name = 'William Tatcher')
+ORDER BY visits.visit_date DESC
+LIMIT 1;
+ 
+--  How many different animals did Stephanie Mendez see?
+
+ SELECT COUNT(DISTINCT animal_id) AS num_animals_seen
+FROM visits
+WHERE vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
+
+-- List all vets and their specialties, including vets with no specialties.
+
+SELECT vets.name AS vet_name, specialties.name AS specialty_name
+FROM vets
```

Comment:
-[ ] Here also, your query is great but we need to see all the columns from the `vests` side instead of the just the `name`. Please use `vets.*, species.name` instead of `vets.name,  species.name`